src/react/features/filters/useFilteredDiorys.js

```diff
@@ -31,9 +31,7 @@ export const useFilteredDiorys = () => {
   const diorys = useDiorys(dioryIds)
   return useMemo(
     () => ({
-      diory: {
-        text: 'Filter',
-      },
+      diory: diorys[0],
```

Comment:
Now when we don't interpolate geo/date to focus diory, we can't give `{ text: 'Filter' }` diory to the map lens.

So quick fix is to have "first linked diory" in focus instead.